p2p/README.md

```diff
@@ -100,34 +115,3 @@ Given a specific `originatorNode` which initiates the broadcast, the `testRainTr
 
 - **Deterministic Private Key Generation**: Since RainTree is dependant on the lexicographic order of the addresses, the generation of the private keys (and in turn the public keys and addresses) is important and cannot be randomized for the time being.
 - **Variable Coupling**:There is an implicit coupling between `validatorId`, `serviceUrl` and `genericParam` that requires understanding of the codebase. Reach out to @olshansk or @andrewnguyen22 for clarity on this.
-
-### Notes
```

Comment:
Can you explain to me the reasoning for removing this?

Reply:
I believe I sent the original PR before it was review for review. Those were just rough "talking notes" from our meeting.

The real delta (relative to main) is this:

![Screen Shot 2022-10-17 at 11 02 20 AM](https://user-images.githubusercontent.com/1892194/196249966-5c268324-2f76-40b9-8b29-7731a98ab7bc.png)